styles.css

```diff
@@ -0,0 +1,19 @@
+body{
+    text-align: center;
+    font-family: Arial;
+    color: blue;
+    max-width: 100%;
+  }
+  h1{
+    font-size: 40px;

```

Comment:
After writing a style there is need to skip a line . this makes your code to look professional

Reply:
- Great Job using Inline comments to highlight lines of code you are suggesting changes on and stating reasons WHY a change needs to be implemented💪🏽 

- I think it will be better to phrase your comment to sound more politely, as suggestions instead of orders.
For example: "I think it would be better to skip a line after writing a style because this will make your code to look professional "
What do your think about how I re phrased your comment?

- Also, I think it will be great to add some emoji's as this will help your comment to sound a bit more friendly 🤗 

Why don't we re phrase all your review comments to what I have suggested above? 🤔